python/lesson02/README.md

```diff
@@ -95,7 +95,8 @@ The label, or `ReferenceType`, describes the nature of the relationship. `ChildO
 means that the `root_span` has a logical dependency on the child `span` before `root_span` can
 complete its operation. Another standard reference type in OpenTracing is `FollowsFrom`, which
 means the `root_span` is the ancestor in the DAG, but it does not depend on the completion of the
-child span, for example if the child represents a best-effort, fire-and-forget cache write.
+child span, for example if the child represents a best-effort, fire-and-forget cache write or
+down-stream processing in an event-driven architecture.
```

Comment:
Hmm this makes it sound more complicated. I think cache write is a cleaner example.